### ISSUE TITLE
Do we need to invalidate the native value holder so that :invalid works?

### ISSUE THREAD

**Issue by mercmobily** [mobily-enterprises/tpe#8]
The `:invalid` selector only ever applies to native elements.
Right now, ee-autocomplete-input-span sets a `valid` attribute for itself, but the `:invalid` selector doesn't work.
It DOES include a native element, mainly to hold the value so that the element can be used in native forms. I make sure to update this value every time something is picked or deleted.
However, I saw the :invalid used in several spots...
I *can* add am invisible native element and set that invalid artificially. If I set it as invalid, I will automatically have the @invalid event fired without making one up artificially (since the native element will generate it and the event will be retargeted)

Would it be useful for you if I added an invisible element artificially set as invalid -- to which you can apply the :invalid selector? Or would it be a total waste of time?

**Comment by freudFlintstone** [comment]
I don't think we need it. If the element set's it's own invalid boolean attribute, css can use :host([invalid]) instead of :invalid. If we make this a documented pattern for all ee elements, I don't see a problem. We could try to make this more consistent across the whole project by avoiding :invalid even in nn elements, though, which would mean setting that invalid attribute in all our elements.

**Comment by mercmobily** [comment]
So, is it all good? Can this be closed?
